PSReadLine/ReadLine.cs

```diff
@@ -206,7 +206,7 @@ internal static PSKeyInfo ReadKey()
                         bool runPipelineForEventProcessing = false;
                         foreach (var sub in eventSubscribers)
                         {
-                            if (sub.SourceIdentifier.Equals("PowerShell.OnIdle", StringComparison.OrdinalIgnoreCase))
+                            if (sub.SourceIdentifier.Equals(PSEngineEvent.OnIdle, StringComparison.OrdinalIgnoreCase))
```

Comment:
The same change applies to line 215.

Reply:
Cannot believe I didn't see that ... Fixed.